python/triton/language/semantic.py

```diff
@@ -554,14 +554,14 @@ def view(input: tl.tensor, dst_shape: List[int], builder: ir.builder) -> tl.tens
     return tl.tensor(builder.create_reshape(input.handle, dst_shape, True), ret_ty)
 
 
-def reshape(input: tl.tensor, dst_shape: List[int], builder: ir.builder) -> tl.tensor:
+def reshape(input: tl.tensor, dst_shape: List[int], reorder: bool, builder: ir.builder) -> tl.tensor:
```

Comment:
Can we name this `can_reorder`, to stay consistent with other ops that have similar hint?